command/workspace_show.go

```diff
@@ -0,0 +1,37 @@
+package command
+
+import (
+	"strings"
+)
+
+type WorkspaceShowCommand struct {
+	Meta
+}
+
+func (c *WorkspaceShowCommand) Run(args []string) int {
+	args = c.Meta.process(args, true)
+
+	cmdFlags := c.Meta.flagSet("workspace show")
+	cmdFlags.Usage = func() { c.Ui.Error(c.Help()) }
+	if err := cmdFlags.Parse(args); err != nil {
+		return 1
+	}
+
+	workspace := c.Workspace()
+	c.Ui.Output(workspace)
+
+	return 1
```

Comment:
Ahh... I didn't catch this on the first pass. We need to `return 0` here to indicate that the command has completed without error. Numbers greater than zero indicate errors, as we can see with the flag parse error above.